pegasus/sites.v3/code.org/styles_min/040-page.css

```diff
@@ -167,40 +165,47 @@ td {
 .listed_supporter{
   font-family: 'Gotham 7r', sans-serif;
   font-weight: normal;
-  color: #7665a0;
 }
 
 .greybox {
-  background-color: rgb(88, 103, 113); color: white; padding: 20px; text-align: center; overflow:hidden;
+  background-color: var(--neutral_dark80);
+  color: var(--neutral_white);
+  padding: 20px;
+  text-align: center;
+  overflow:hidden;
 }
 
 #about_nav {
   margin: 50px 0 0 0;
 }
 
 .about_list_item_selected {
-  color: #ff8a00;
+  color: var(--brand_secondary_default);
   font-family: 'Gotham 7r', sans-serif;
 }
 
 a.about_list_item {
-  color: #999c9f;
+  color: #7F8286;
```

Comment:
nit: looks like you've missed this one (--neutral_dark60)

Reply:
Great catch, ty!